tools/integration_tests/operations/copy_dir_test.go

```diff
@@ -242,17 +241,16 @@ func checkIfCopiedEmptyDirectoryHasNoData(destSrc string, t *testing.T) {
 // destNonEmptyCopyDirectory/subDirInNonEmptyDestCopyDirectory
 // destNonEmptyCopyDirectory/emptySrcDirectoryCopyTest
 func TestCopyEmptyDirectoryInNonEmptyDirectory(t *testing.T) {
-	// Clean the mountedDirectory before running test.
-	setup.CleanMntDir()
+	testDir := setup.SetupTestDirectory(DirForOperationTests)
 
-	srcDir := path.Join(setup.MntDir(), EmptySrcDirectoryCopyTest)
+	srcDir := path.Join(testDir, EmptySrcDirectoryCopyTest)
 	operations.CreateDirectoryWithNFiles(0, srcDir, "", t)
 
 	// Create below directory
 	// destNonEmptyCopyDirectory                                                -- Dir
 	// destNonEmptyCopyDirectory/subDirInNonEmptyDestCopyDirectory              -- Dir
-	destDir := path.Join(setup.MntDir(), DestNonEmptyCopyDirectory)
-	createDestNonEmptyDirectory(t)
+	destDir := path.Join(testDir, DestNonEmptyCopyDirectory)
+	createDestNonEmptyDirectory(destDir, t)
```

Comment:
Nit, can we merge line 252 and 253 into the same function. You can return dirPath from the create call for ease of readability